ops-bedrock/docker-compose.yml

```diff
@@ -12,6 +12,9 @@ volumes:
   da_data:
   op_log:
 
+networks:
```

Comment:
can we add some comments here that this is needed to Babylon e2e test/deployment script?

this will be helpful when later we decided to submit a PR to the OP foundation to get it merged. b/c we can then remove all the irrelevant changes.

Reply:
Sure.
I tried to update it with the bash script (`sed` and [yq](https://github.com/mikefarah/yq)) before the op devnet setup (and when devnet down, revert this file), but the generated file is not what I wanted (although it was updated with needed content, the whole file is formatted by `yq`).

Reply:
ah got it. nice i haven't' heard of yq but seems useful